cl_showpos/Windows/ShowposWindow.cs

```diff
@@ -43,10 +43,11 @@ public override void PreDraw() {
         var viewport = ImGui.GetMainViewport();
         var windowPos = viewport.Pos;
         var screenSize = viewport.Size;
+        var topSafeArea = Plugin.PluginInterface.IsDevMenuOpen ? (ImGuiHelpers.GlobalScale * 20) : 0;
```

Comment:
I think I used `ImGui.FrameHeight()` to do a similar thing to this once, so maybe see if that works instead of this magic constant? I'm not 100% sure though.

Reply:
Yup that works even better, and it's already affected by scaling!